docs/userGuide/siteJsonFile.md

```diff
@@ -33,9 +33,11 @@ Here is a typical `site.json` file:
       }
     },
     {
-      "glob": "**/index.md"
+      "glob": "**/index.md",
+      "globExclude": ["internal/index.md"]
     }
   ],
+  "pagesExclude": ["admin/index.md"],
```

Comment:
should we make the example a little more realistic? 

```
"pagesExclude": ["subsite/*.md"]
```

if possible also the above one as well

Reply:
Ahh yes that would be better

Reply:
I could change the `globExclude` value to `subsite/**/index.md` and the `pagesExclude` to `subsite/*.md`, would that be okay?

Reply:
hmm I used `subsite` in `pagesExclude` because you would normally only want to reuse all files in the subsite, not deploy them as part of the main site. (it should be `subsite/**.md` as well my mistake)

I was thinking something like this (barring the generic `topic1` 🙈):

```
glob: "topic1/**.md",
globExclude: ["topic1/appendix_*.md"], (not sure if this a valid glob :eyes:)
layout: "subtopic"
```

where the benefit of `globExclude` becomes clear as opposed to `pagesExclude`

Reply:
Ohh I see what you mean now. I was trying to find other general names for the example and just couldn't find any other than `subsite` 😓 I think the above is quite nice, I might change it a bit to be slightly more generalized (following the "topic" illustration):

```
glob: "topics/*/*.md",
globExclude: ["topics/*/appendix/*.md"],
layout: "subtopic"
```

With `pagesExclude` still contains `subsite/*/*.md` (I think this is proper glob already, so actually you didn't do any mistakes haha).

Reply:
looks good! 👍

Reply:
👌 I have pushed a commit for the change